### PR TITLE
refactor: upgrading gatsby version from 4.9.0 to 4.14.1

### PR DESCRIPTION
**Pull request by ashraf-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,22 +15,22 @@
...
         "formik": "^2.2.9",
-        "gatsby": "^4.9.3",
+        "gatsby": "^4.14.1",
         "gatsby-background-image": "^1.6.0",
         "gatsby-plugin-anchor-links": "^1.2.1",
-        "gatsby-plugin-google-tagmanager": "^4.9.0",
-        "gatsby-plugin-image": "^2.9.1",
-        "gatsby-plugin-manifest": "^4.9.1",
+        "gatsby-plugin-google-tagmanager": "^4.14.0",
+        "gatsby-plugin-image": "^2.14.1",
```
Comment (mohammad-rework) [inline package.json:22]: We will still get `.innerHTML` error with this version of `gatsby-plugin-image`.
For checking this you can go to /dmt5 page and see the console.

Comment (habib-deriv) [reply package.json:22]: This issue is resolved in this
"gatsby-plugin-image": "2.13.0",

Comment (ashraf-deriv) [reply package.json]: Thanks guys, downgraded  gatsby-plugin-image.

**Review by yashim-deriv — APPROVED** [review]

**Review by yashim-deriv — DISMISSED** [review]

**Review by prince-deriv — APPROVED** [review]

**Comment by ashraf-deriv** [comment]
> Are we still using Node v14.* here?

we are updating to V16 by this PR.

**Review by ali-hosseini-fs — APPROVED** [review]
LGTM

**Review by mohammad-rework — APPROVED** [review]

**Comment by michio-rework** [comment]
Please update your node verion to node >=16.6.0, npm >=8.11.0. and install the packages as well.
and resolve conflicts with it, thanks thanks :pray:

**Comment by ashraf-deriv** [comment]
> 

Hi @michio-rework , It is done . Please review and merge, Thanks 🙏

**Comment by michio-rework** [comment]
> > 
> 
> Hi @michio-rework , It is done . Please review and merge, Thanks pray

Hi @ashraf-deriv , 
There are changes on translations, your PR does not contain any translation changes. can you please take a look at it? 
I think if you merge master into this it will be resolved.

**Comment by ashraf-deriv** [comment]
> > > 
> > 
> > 
> > Hi @michio-rework , It is done . Please review and merge, Thanks pray
> 
> Hi @ashraf-deriv , There are changes on translations, your PR does not contain any translation changes. can you please take a look at it? I think if you merge master into this it will be resolved.

Hi @michio-rework , Actually I haven't added any changes on translation, It is because of prettify and it is automatically came in. I think it is fine to merge. And as you said I have got the latest pull from master and push to the PR as well.

**Review by michio-rework — DISMISSED** [review]

**Review by mohammad-rework — APPROVED** [review]

**Pull request merged**